conf/erda-configs/permission/backend/sys/org/org.yml

```diff
@@ -18,6 +18,10 @@
   scope: org
   resource: cluster
   action: LIST
+- role: Manager,Dev,Ops,Support,DataEngineer,Guest
```

Comment:
In what scenarios does a guest need to view cluster information?

Reply:
For some public organizations users may be allowed to access their cluster data. 
Like `autotest`